### PR TITLE
:bug: CAPM3 local image registry path faulty

### PR DESCRIPTION
**Pull request by mikkosest**

### CODE REVIEW

File: 03_launch_mgmt_cluster.sh [hunk 1]
```diff
@@ -104,8 +104,8 @@ function clone_repos() {
 function patch_clusterctl(){
   pushd "${CAPM3PATH}"
   if [ -n "${CAPM3_LOCAL_IMAGE}" ]; then
-    export MANIFEST_IMG="192.168.111.1:5000/localimages/cluster-api-provider-metal3"
-    export MANIFEST_TAG="${CAPM3RELEASE}"
+    export MANIFEST_IMG="192.168.111.1:5000/localimages/tested_repo"
```
Comment (maelk) [inline 03_launch_mgmt_cluster.sh:107]: Isn't that the CAPM3_LOCAL_IMAGE ?

Comment (maelk) [reply 03_launch_mgmt_cluster.sh:107]: Then you would be better off using something like
```
IMAGE_NAME="${CAPM3_LOCAL_IMAGE##*/}:latest"
LOCAL_IMAGE="192.168.111.1:5000/localimages/$IMAGE_NAME"
```

Comment (mikkosest) [reply 03_launch_mgmt_cluster.sh]: done

**Pull request merged**